app/components/carousel/Carousel.tsx

```diff
@@ -103,7 +103,7 @@ export function Carousel(props: CarouselProps) {
               {...{
                 imageStyle: (item as DynamicCarouselItem).imageStyle,
                 index,
-                isSpeakerPanel: (item as DynamicCarouselItem).isSpeakerPanel,
+                speakerPanelOrWorkshop: (item as DynamicCarouselItem).speakerPanelOrWorkshop,
```

Comment:
So I've been thinking through this, and what would make the most sense to me is for the `CarouselCard` to have a `variant` or `preset` prop. Then that would either be `default` or `speaker`. Then you use that to apply the logic within `CarouselCard`. Then instead of a boolean, `Carousel` would have a prop called `carouselCardVariant` or `carouselCardPreset` which accepts the type of variant. That way you don't have to do so much renaming in the future, you just add a new variant.